pkg/reporting/reporting_test.go

```diff
@@ -66,6 +66,9 @@ func TestReportingWirhOverrides(t *testing.T) {
 
 // NOTE: @afiune we don't report an error if we were unable to load the config.toml
 func TestReportingNewDefaultErrorWithoutCredentials(t *testing.T) {
+	// NOTE - this fails if you have a local credential set configured.
+	//        here (and elsewhere) we should prevent filesystem access in
+	//        unit tests. This looks promising: https://github.com/spf13/afero
```

Comment:
oh right, since I am running everything from within a Hab Studio, I don't have that problem, but I can see why that would fail. Exploring the afero package seems like a good idea!

Reply:
I opened this yesterday to track  https://github.com/chef/chef-analyze/issues/59